dhall-to-cabal.dhall

```diff
@@ -90,6 +90,7 @@ in    prelude.utils.GitHub-project
           "Distribution"
       -- build-type simple is needed to allow tools compatible with cabal < 2.2 build the package
       , build-type =
+          Some ( types.BuildType.Simple {=} )
           [ types.BuildType.Simple {=} ] : Optional types.BuildType
```

Comment:
This doesn't look right.

Reply:
oops, bad merge, thanks for pointing out

Reply:
Corrected, and tested with `dhall-to-cabal dhall-to-cabal.dhall`